papaparse.js

```diff
@@ -511,6 +511,11 @@ License: MIT
 		this.parseChunk = function(chunk, isFakeChunk)
 		{
 			// First chunk pre-processing
+			const skipFirstNLines = parseInt(this._config.skipFirstNLines) || 0;
+			if (this.isFirstChunk && skipFirstNLines > 0) {
+				const splitChunk = chunk.split('\n');
```

Comment:
Please do not hardcode the line break, there is a config setter which defines the valid line breaks and allows the user to change them. Code should work for any of them.

Reply:
Good point! I changed the code. Unfortunately, made it a bit more complicated because config.newline might be undefined. I used the same approach like [here](https://github.com/mholt/PapaParse/pull/1046/files#diff-fbb8d0b19dc4586fadd80f52a7b49415d2d41a5fdccaeaa728f7fca51cfbbb29R1090)